install.m

```diff
@@ -0,0 +1,15 @@
+% This is installation file for FEX
+
+% Determine where this m-file's folder is.
+[base_dir,~,~] = fileparts(mfilename('fullpath'));
+
+% Add PhysicalQuantity folder plus all subfolders to the path.
+PQ_path     = [base_dir  '\PhysicalQuantity'        ];

```

Comment:
This is Windows-specific; the backslash will break on Linux/Mac. 
Use `fullfile(base_dir, 'PhysicalQuantity')`